.tx/config

```diff
@@ -0,0 +1,9 @@
+[main]
+host = https://www.transifex.com
+
+[foreman.hammer-cli-foreman-docker]
+file_filter = locale/<lang>/hammer-cli-foreman-docker.po
```

Comment:
Your PO/POT files have underscores, but this uses hyphens.  For hammer..discovery, I think @orrabin stuck with underscores in the end.